lib/logstash/inputs/jdbc.rb

```diff
@@ -2,7 +2,7 @@
 require "logstash/inputs/base"
 require "logstash/namespace"
 require "logstash/plugin_mixins/jdbc/jdbc"
-
+require_relative "tzinfo_jruby_patch"
```

Comment:
related to my other comment, if this is a required monkey patch, is should be guarded by version checks to make sure it does not blow up in a future update.

Reply:
Do you mean guard on the JRuby version > 9.0.0.0 and < 9.2.8.0?
Headius tagged the issue with 9.2.8.0 but there is no guarantee.